dev/kinto.ini

```diff
@@ -0,0 +1,54 @@
+[app:main]
+use = egg:kinto
+kinto.includes = kinto_changes kinto_attachment
+
+kinto.storage_backend = kinto.core.storage.memory
+kinto.storage_url =
+kinto.cache_backend = kinto.core.cache.memory
+kinto.cache_url =
+kinto.permission_backend = kinto.core.permission.memory
+kinto.permission_url =
+
+kinto.bucket_create_principals = system.Everyone
+kinto.bucket_read_principals = system.Everyone
+kinto.bucket_write_principals = system.Everyone
```

Comment:
Nice :) I don't think I ever tried that! I always configure authentication :smile: